src/ape/api/networks.py

```diff
@@ -34,40 +32,58 @@ class EcosystemAPI:
     The name of the ecosystem. This should be set the same name as the plugin.
     """
 
-    network_manager: "NetworkManager"
-    """A reference to the global network manager."""
-
-    config_manager: "ConfigManager"
-    """A reference to the global config manager."""
-
-    plugin_manager: PluginManager
-    """A reference to the global plugin manager."""
-
     data_folder: Path
     """The path to the ``.ape`` directory."""
 
-    request_header: str
+    request_header: dict
     """A shareable HTTP header for network requests."""
 
-    transaction_types: Dict["TransactionType", Type["TransactionAPI"]]
-    """The available types of transaction API this ecosystem supports."""
+    _default_network: str = LOCAL_NETWORK_NAME
 
-    receipt_class: Type["ReceiptAPI"]
-    """The receipt class for this ecosystem."""
+    @abstractmethod
+    def serialize_transaction(self, transaction: "TransactionAPI") -> bytes:
+        """
+        Serialize a transaction to bytes.
 
-    block_class: Type["BlockAPI"]
-    """The block class for this ecosystem."""
+        Args:
+            transaction (:class:`~ape.api.TransactionAPI`): The transaction to encode.
```

Comment:
You have to link the full path unfortunately

```suggestion
            transaction (:class:`~ape.api.providers.TransactionAPI`): The transaction to encode.
```